synapseclient/models/dataset.py

```diff
@@ -2573,37 +2339,32 @@ def to_synapse_request(self):
         delete_none_keys(result)
         return result
 
-    async def add_item_async(
+    def add_item(
         self,
-        item: Union["Dataset", "EntityRef"],
+        item: Union["Dataset"],
```

Comment:
Is `EntityRef` still accepted here? it is used within the function and exception message